ext/nnstreamer/include/nnstreamer.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package NNStreamer;
+
+message Tensor {
+  string name = 1;
+  enum Tensor_type {
+    NNS_INT32 = 0;
+    NNS_UINT32 = 1;
+    NNS_INT16 = 2;
+    NNS_UINT16 = 3;
+    NNS_INT8 = 4;
+    NNS_UINT8 = 5;
+    NNS_FLOAT64 = 6;
+    NNS_FLOAT32 = 7;
+    NNS_INT64 = 8;
+    NNS_UINT64 = 9;
+  
+    NNS_END = 10;
```

Comment:
Do we need to define NNS_END in proto? (in C header, END is for loop-break. but I'm not sure if we need this for proto)

We may have additional types defined later and keep it compatible with older types.

Reply:
I remove the `NNS_END`. The field number of additional types could be assigned with # starting from 10.

ref: https://developers.google.com/protocol-buffers/docs/proto3?#updating